nixos/modules/services/backup/restic.nix

```diff
@@ -170,11 +223,22 @@ in
             ( resticCmd + " forget --prune " + (concatStringsSep " " backup.pruneOpts) )
             ( resticCmd + " check" )
           ];
+          # Helper functions for rclone remotes
+          isRcloneRemote = hasPrefix "rclone:";
```

Comment:
This is not used anywhere?

Reply:
Oops, that's a leftover from before I wrapped restic with rclone. Taking that out.